src/http/controllers/core.controller.ts

```diff
@@ -31,6 +33,19 @@ export class CoreController {
         }
     }
 
+    @Get('stats')
+    async stat() {
+        const lumClt = await this._lumNetworkService.getClient();
+
+        const [inflation] = await Promise.all([lumClt.queryClient.mint.inflation().catch(() => null)]);
```

Comment:
I believe this could be improved and kept as a single promise returning a single value

Reply:
This is a stats endpoint. For now, we have inflation only, but we will have many other stats soon. If this can explain the `Promise.all()`

If anything fail, the inflation card will not be displayed in explorer

Reply:
Explorer is not the only potential usecase of the bridge :)

Reply:
I added a default value